packages/language-server/src/codeActionProvider.ts

```diff
@@ -258,6 +258,32 @@ export function quickFix(
         },
       })
     }
+    if (diag.severity === DiagnosticSeverity.Error &&
+    diag.message.includes('It does not start with any known Prisma schema keyword.')) {
+      let diagText = textDocument.getText(diag.range).split(/\s/)
+      const spellingSuggestion = getSpellingSuggestions(
+        diagText[0],
```

Comment:
Nit: Maybe move the `[0]` to line 263. Can this be `null`? If yes, does the function `getSpellingSuggestions` handle null well?